Watch Extension/TrendingInterfaceController.swift

```diff
@@ -28,18 +28,18 @@ class TrendingInterfaceController: WKInterfaceController {
             self.loadingImageView.setHidden(true)
         }
     }
-    
+
     func fetchTrendingNews() {
         showLoadingIndicator(true)
-        _ = newsService.requestTrendingTopicsWithDate(Date(), count: 6, success: { (result) in
+        _ = newsService.requestTrendingTopicsWithDate(Date(), count: 6, success: { result in
             self.topics = result?.topics
-            
+
             guard let topics = self.topics else {
                 return
             }
-            
+
             self.trendingTable.setNumberOfRows(topics.count, withRowType: "TrendingRow")
-            
+
             for (index, t) in topics.enumerated() {
                 let row = self.trendingTable.rowController(at: index) as! TrendingRowController
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/f5a5ab936ee0d164c160af7374b637afef6eb821/Watch Extension/TrendingInterfaceController.swift#L44"></span>Force casts should be avoided.<br />
<code>force_cast</code> <code>TrendingInterfaceController.swift:44</code></td>
    </tr>
  </tbody>
</table>